src/model.d.ts

```diff
@@ -1181,21 +1178,29 @@ export interface BulkCreateOptions<TAttributes = any> extends Logging, Transacti
    */
   ignoreDuplicates?: boolean;
 
+  /**
+   * Return all columns or only the specified columns for the affected rows (only for postgres)
+   */
+  returning?: boolean | Array<keyof TAttributes | Literal | Col>;
+
   /**
    * Fields to update if row key already exists (on duplicate key update)? (only supported by MySQL,
    * MariaDB, SQLite >= 3.24.0 & Postgres >= 9.5).
    */
   updateOnDuplicate?: Array<keyof TAttributes>;
 
   /**
-   * Include options. See `find` for details
+   * Primary key names used as conflict target columns during insertion.
    */
-  include?: AllowArray<Includeable>;
+  upsertKeys?: string[];
```

Comment:
Can you undo the changes to the order so only `upsertKeys` is added?

Reply:
👍 sure!

Reply:
`yarn lint` changed the orders of the `ModelHooks` imports but the pipeline is not passing the lint checks that pass locally. 
Am I missing some steps in the PR submission process?

Reply:
`yarn lint` should not do this (the CI is running `yarn lint-no-fix` with the same rules), but you are able to undo these changes manually